src/Effects.js

```diff
@@ -472,3 +461,138 @@ module.exports = class Effects {
     };
   }
 };
+
+function drawSwirl(ctx) {
+  ctx.fillStyle = "#333";
```

Comment:
Should each of these be wrapped in `ctx.push()` and `ctx.pop()` calls to ensure we don't bleed drawing state into other operations?

Reply:
Never mind, I see this is part of the `draw` method above.